src/algorithm/MEMCPY-OMPTarget.cpp

```diff
@@ -29,61 +29,63 @@ namespace algorithm
 #define MEMCPY_DATA_SETUP_OMP_TARGET \
   int hid = omp_get_initial_device(); \
   int did = omp_get_default_device(); \
-\
-  allocAndInitOpenMPDeviceData(x, m_x, iend, did, hid);
+                                      \
+  allocAndInitOpenMPDeviceData(x, m_x, iend, did, hid); \
+  allocAndInitOpenMPDeviceData(y, m_y, iend, did, hid);
 
-#define MEMCPY_DATA_TEARDOWN_OMP_TARGET \
-  deallocOpenMPDeviceData(x, did); \
+#define MEMCPY_DATA_TEARDOWN_OMP_TARGET                                        \
+  deallocOpenMPDeviceData(x, did);                                             \
+  deallocOpenMPDeviceData(y, did);
 
 
 void MEMCPY::runOpenMPTargetVariant(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune_idx))
 {
-  const Index_type run_reps = getRunReps();
-  const Index_type ibegin = 0;
-  const Index_type iend = getActualProblemSize();
+    const Index_type run_reps = getRunReps();
```

Comment:
Please don't change the format of the code unless it improves readability. Our convention is indent two-spaces, no tabs in source code.

Reply:
Yes, sorry. I have undone the formatting and will upload 'cleaned' files. That's why I marked the PR as draft.